Makefile

```diff
@@ -22,8 +22,5 @@ test-unit:
 test-deprecation:
 	$(PYTEST) -Werror::DeprecationWarning -Werror::FutureWarning ./test/unit
 
-doc-multiversion:
-	python scripts/build_documentation.py --documentation-path=docs --output-path=docs/_build/html
```

Comment:
Can we also delete `build_documentation.py` ?

Reply:
done